### PR TITLE
Home tesk: HTML 07

### PR DESCRIPTION
**Pull request by Vecherskiys**
Please review.
Thank you :)

### CODE REVIEW

File: Vecherskiys/07/Task7.3.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        <META charset="utf-8"> 
+        <TITLE>Ссылки. Оформление</TITLE> 
+		<style type="text/css">
+            table {border-collapse: collapse;}
+            td {
+                width: 50px;
+                height: 50px;
+                border: 1px solid black;
+            }
+            tr:nth-child(odd) td:nth-child(even){background-color: black}
```
Comment (pluhin) [inline Vecherskiys/07/Task7.3.html]: You can combine, remember about comma :)

File: Vecherskiys/07/Task7.4.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		<style type="text/css">
+            table {border-collapse: collapse;}
+            td {
+                width: 50px;
+                height: 50px;
+                border: 1px solid black;
+                text-align: center;
+                font-size: 35px;
+            }
+            tr:nth-child(odd) td:nth-child(even){background-color: black; color: white;}
```
Comment (pluhin) [inline Vecherskiys/07/Task7.4.html]: here also can be short (combine selectors by comma)

File: Vecherskiys/07/Task7.3.html (hunk 1 above)
Comment (Vecherskiys) [reply Vecherskiys/07/Task7.3.html]: Ok. Complete:)

File: Vecherskiys/07/Task7.4.html (hunk 2 above)
Comment (Vecherskiys) [reply Vecherskiys/07/Task7.4.html]: Ok. Complete:)

**Pull request merged**